oled.js

```diff
@@ -125,10 +126,9 @@ Oled.prototype._transfer = function(type, val, fn) {
 
 // read a byte from the oled
 Oled.prototype._readI2C = function(fn) {
-  this.wire.i2cRead(this.ADDRESS, 0, new Buffer([0]), function(err, bytesRead, data) {
-    // result is single byte
-    fn(data[0]);
-  });
+  var data=[0];
+  this.wire.i2cReadSync(this.ADDRESS, 1, Buffer.from(data));
```

Comment:
Why did you change i2cReadrequest to i2cReadSync?

Please make this line compatible with older version of Node.js